src/util/voxel_io.cpp

```diff
@@ -152,26 +164,33 @@ from_qb(
                 voxels_read++;
             }
 
-    LOG_INFO(logger, "Voxels read: {}", voxels_read);
+    LOG_INFO(logging::file_io_logger, "Voxels read: {}", voxels_read);
 }
 
 // This is partially from goxel with GPL license
 void
-to_qb(const std::string path, terrain::Terrain ter, bool compression)
+to_qb(const std::filesystem::path path, terrain::Terrain ter, bool compression)
 {
-    static quill::Logger* logger = logging::get_logger("voxel_io");
+    // static quill::Logger* logger = logging::get_logger("voxel_io");
 
-    LOG_INFO(logger, "Saving voxels to {}", path);
+    LOG_INFO(
+        logging::file_io_logger, "Saving voxels to {}.",
+        path.lexically_normal().string()
+    );
 
     if (compression) {
-        LOG_ERROR(logger, "Cannot write voxel files with compression");
+        LOG_ERROR(logging::file_io_logger, "Cannot write voxel files with compression");
         throw exc::not_implemented_error("Cannot write compressed voxel files");
     }
 
     // Saves the tiles in this to the path specified
     std::ofstream file(path, std::ios::out | std::ios::binary);
     if (!file) {
-        LOG_ERROR(logger, "Could not open {}. Are you in the right directory?", path);
+        LOG_ERROR(
+            logging::file_io_logger,
+            "Could not open {}. Are you in the right directory?",
+            path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.